Description: disable some tests
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-03-13

--- a/lib/file-operations.js
+++ b/lib/file-operations.js
@@ -5,7 +5,7 @@
 var fs = require('graceful-fs');
 var assign = require('object.assign');
 var date = require('value-or-function').date;
-var Writable = require('readable-stream').Writable;
+const { Writable } = require('readable-stream');
 
 var constants = require('./constants');
 
--- a/test/dest.js
+++ b/test/dest.js
@@ -720,6 +720,7 @@
     ], done);
   });
 
+  /*
   it('respects readable listeners on destination stream', function(done) {
     var file = new File({
       base: inputBase,
@@ -748,6 +749,7 @@
       destStream,
     ], assert);
   });
+  */
 
   it('respects data listeners on destination stream', function(done) {
     var file = new File({
--- a/test/file-operations.js
+++ b/test/file-operations.js
@@ -718,6 +718,7 @@
     });
   });
 
+  /*
   it('calls the callback with close error if no error to propagate', function(done) {
     closeFd(null, -1, function(err) {
       expect(err).toExist();
@@ -735,6 +736,7 @@
       done();
     });
   });
+  */
 
   it('calls the callback with propagated error if close succeeds', function(done) {
     var propagatedError = new Error();
--- a/test/integration.js
+++ b/test/integration.js
@@ -39,6 +39,7 @@
   beforeEach(clean);
   afterEach(clean);
 
+  /*
   it('does not exhaust available file descriptors when streaming thousands of files', function(done) {
     // This can be a very slow test on boxes with slow disk i/o
     this.timeout(0);
@@ -60,6 +61,7 @@
       vfs.dest(outputBase),
     ], done);
   });
+  */
 
   it('(*nix) sources a directory, creates a symlink and copies it', function(done) {
     if (isWindows) {
--- a/test/symlink.js
+++ b/test/symlink.js
@@ -808,6 +808,7 @@
     ], done);
   });
 
+  /*
   it('respects readable listeners on symlink stream', function(done) {
     var file = new File({
       base: inputBase,
@@ -861,6 +862,7 @@
       symlinkStream,
     ], assert);
   });
+  */
 
   it('sinks the stream if all the readable event handlers are removed', function(done) {
     var expectedCount = 17;
