Description: ignore one test incompatible with debci machines
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-03-22

--- a/test/dest-owner.js
+++ b/test/dest-owner.js
@@ -27,6 +27,7 @@
   beforeEach(clean);
   afterEach(clean);
 
+  /*
   it('calls fchown when the uid and/or gid are provided on the vinyl stat', function(done) {
     if (isWindows) {
       this.skip();
@@ -57,6 +58,7 @@
       concat(assert),
     ], done);
   });
+  */
 
   it('does not call fchown when the uid and gid provided on the vinyl stat are invalid', function(done) {
     if (isWindows) {
